pkg/util/mount/nsenter_mount.go

```diff
@@ -170,6 +170,12 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		return true, err
 	}
 
+	// Check the directory exists
+	if _, err = os.Stat(file); os.IsNotExist(err) {
```

Comment:
yes, it's `/var/lib/kubelet`, sorry.

@fgrzadkowski, I don't understand your point. If `var/lib/kubelet/test/foo` exist, this patch does not change a thing. Only if it doesn't we return `ErrNotExist` to tell the caller to create the directory before mounting.